apps/test/unit/templates/teacherDashboard/OwnedSectionsTest.js

```diff
@@ -100,7 +100,7 @@ describe('OwnedSections', () => {
     );
     expect(spy).not.to.have.been.called;
 
-    expect(wrapper.find('Notification').length).to.equal(1);
-    expect(wrapper.find('Notification').props().notice).to.equal('Add a new classroom section');
+    expect(wrapper.find('Connect(Notification)').length).to.equal(1);
+    expect(wrapper.find('Connect(Notification)').props().notice).to.equal('Add a new classroom section');
```

Comment:
Just a note: If you don't like mentioning `Connect(*)` in tests like this, you can always import the actual component and use that to query the wrapper.

```jsx
import Notification from 'wherever';
expect(wrapper.find(Notification).length).to.equal(1);
```